frontend/src/component/project/Project/ProjectLog/ProjectLog.tsx

```diff
@@ -0,0 +1,21 @@
+import { EventLog } from 'component/events/EventLog/EventLog';
+import { useRequiredPathParam } from 'hooks/useRequiredPathParam';
+import { useStyles } from './ProjectLog.styles';
+
+const ProjectLog = () => {
+    const projectId = useRequiredPathParam('projectId');
+
+    const { classes: styles } = useStyles();
+
+    return (
+        <div className={styles.container}>
+            <EventLog
+                title="Event Log"
+                project={projectId}
+                displayInline
+            ></EventLog>
+        </div>
+    );
+};
+
+export default ProjectLog;
```

Comment:
We do not use default export since it has a high potential for bugs and confusion, all exports should be named exports.

Reply:
Removed the usage of `useStyles` in favour of a styled component